File: C++/16.cpp
#include <iostream>

using namespace std;

int main()
{

    int size, sum = 0, count = 0;

    cout << "Enter the number of elements: ";
    cin >> size;

    int list[size];

    cout << "Enter elements of the lis: " << endl;

    for (int i = 0; i < size; i++)
        cin >> list[i];

    for (int i = 0; i < size; i++)
    {
        sum = sum + list[i];
        count++;
    }

    cout << "Arithmetic mean of the elements of the list: " << sum / count;
}

File: Python/16.py
myList = []
sumOfElements = 0
counter = 0

numElements = int(input("How many elements would you like to add? "))

for i in range(numElements):
    element = int(input("Enter an element: "))
    myList.append(element)



for i in range(numElements):
    sumOfElements += myList[i]
    counter += 1

print("Arithmetic mean of the elements of the list: ", sumOfElements / counter)
